script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -357,17 +357,17 @@ The officers raise their pistols and fire in unison.
 [sha:b2e0bbb2bbbff3b8219417947d286c828a48ad33]{
 -- Page 400, Offset 9636.
 -- だがヤツは微動だにしない。
-However, the man does not budge.
+However, he doesn't so much as lift a finger.
 }
 [sha:618804708ab01d0be5b8f6599b6f86eaaf7b337a]{
 -- Page 400, Offset 9637.
 -- 警官たちの事なんて、きっと気がついてもいない。
-The cops most likely don't even register to him.
+He likely didn't even fathom their presence.
```

Comment:
Fathom feels like the wrong word here - it's not that he doesn't understand, it's that he doesn't notice